### PR TITLE
Fix assorted command handlers

### PR DESCRIPTION
**Pull request by jerith**

### CODE REVIEW

File: go/apps/bulk_message/vumi_app.py [hunk 1]
```diff
@@ -126,6 +126,11 @@ def process_command_send_message(self, user_account_key, conversation_key,
         if kwargs:
             log.info("Received unexpected command args: %s" % (kwargs,))
         conv = yield self.get_conversation(user_account_key, conversation_key)
+        if conv is None:
+            log.warning("Cannot find conversation '%s' for user '%s'." % (
+                user_account_key, conversation_key))
```
Comment (justinvdm) [inline go/apps/bulk_message/vumi_app.py]: Should this tuple be `(conversation_key, user_account_key)`?

Comment (justinvdm) [inline go/apps/bulk_message/vumi_app.py]: Similarly for a few of the other logged warning additions in this PR

Comment (jerith) [inline go/apps/bulk_message/vumi_app.py]: Good catch, thanks. Too much copy/paste.

**Comment by justinvdm** [comment]
:+1:

**Pull request merged**